assets/js/base/utils/get-inline-styles.ts

```diff
@@ -5,15 +5,12 @@ import classnames from 'classnames';
 import { paramCase as kebabCase } from 'change-case';
 import { getCSSRules } from '@wordpress/style-engine';
 import { isObject } from '@woocommerce/types';
+import type { Style as StyleEngineProperties } from '@wordpress/style-engine/src/types';
+import type { CSSProperties } from 'react';
 
-type StyleValue = string | NestedStyle;
-
-interface NestedStyle {
-	[ key: string ]: StyleValue | undefined;
-}
-
-export type WithStyle = {
-	style: Record< string, NestedStyle > | undefined;
+export type StyleProps = {
```

Comment:
Looks like this specific export is not used, there is one here but not exported:

https://github.com/woocommerce/woocommerce-blocks/pull/9483/files#diff-7ea7e1b7344c17a19ca399a95e666922812b4a434942fac02bd7a80f3edbd115L23

And another one here: https://github.com/woocommerce/woocommerce-blocks/pull/9483/files#diff-824ff3e98bd52e5879d9ee114831a9f26022945ea4fb2063a8401e84f18157f4R16

Could we just use one of them? I think exporting the one in `use-style-props.ts` might make most sense, but we could also move it to the `types` folder.